sonar-iac-plugin/src/main/java/org/sonar/plugins/iac/IacPlugin.java

```diff
@@ -35,8 +37,12 @@
 
 public class IacPlugin implements Plugin {
 
+  private static final Logger LOG = LoggerFactory.getLogger(IacPlugin.class);
+
+
   @Override
   public void define(Context context) {
+    LOG.info("AAA IacPlugin define");
```

Comment:
Is this some leftover that should be removed? :D

Reply:
🤦‍♂️